src/amp/components/Ticker.tsx

```diff
@@ -1,85 +0,0 @@
-import React from 'react';
-import { css } from 'emotion';
-
-import { headline, body } from '@guardian/src-foundations/typography';
-import { neutral, brandAlt } from '@guardian/src-foundations/palette';
-
-const tickerWrapperStyle = css`
-    margin-bottom: 20px;
-`;
-const leftStyle = css`
-    text-align: left;
-    display: flex;
-    flex-wrap: nowrap;
-    flex-direction: column;
-    justify-content: flex-end;
-`;
-const rightStyle = css`
-    text-align: right;
-    display: flex;
-    flex-wrap: nowrap;
-    flex-direction: column;
-    justify-content: flex-end;
-`;
-const tickerInfoStyle = css`
-    display: flex;
-    flex-wrap: nowrap;
-    flex-direction: row;
-    justify-content: space-between;
-`;
-const tickerProgressStyle = css`
-    position: absolute;
-    margin: 0;
-    padding: 0;
-    left: 0;
-    background-color: ${brandAlt[400]};
-    height: 100%;
-`;
-const tickerBackgroundStyle = css`
-    position: relative;
-    margin: 5px 0;
-    height: 10px;
-    width: 100%;
-    background-color: ${neutral[86]};
-    border: none;
-`;
-const currentAmountStyle = css`
-    ${headline.xsmall({ fontWeight: 'bold' })}
-`;
-const goalAmountStyle = css`
-    ${headline.xxxsmall({ fontWeight: 'bold' })}
-`;
-const amountCaptionStyle = css`
-    ${body.small({ fontStyle: 'italic' })};
-`;
-
-export const Ticker: React.FC<{
```

Comment:
is there a reason for moving this into the other file?

Reply:
It's going to be necessary to do this when we implement the animation. It's not vital at the moment, but I think is probably preferable to have a single component for the epic (including the ticker) as all the data comes from the same HTTP request and populates the same moustache template.